### PR TITLE
Added limitations of Aggregate

### LINKED ISSUE #26: Document Aggregate limitations
Some good ones on this issue - https://github.com/opendatakit/briefcase/pull/142#issuecomment-315474501 and I'm sure there are more on the dev wiki

### PR DESCRIPTION
**Pull request by ankita240796**
closes #26 

## What is included in this PR?
Added limitations of Aggregate and issues with different versions.

### CODE REVIEW

File: aggregate-limitations.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+- Aggregate 0.9.x has a 1000-record limit to the number of records it can display (this is by design); it does, however, still retain all records i.e. it will hold as many records as you've uploaded and does not loose or ignore the 1001st record, etc; in general, any webserver will have a limit to what it can display interactively (at some point it will run out of memory or time out). 
+
+.. tip::
+  
+  - On Aggregate 0.9.x, we provide the Briefcase applet (one of the links on the top row) to allow you to extract your data from the server into a local CSV file for local processing. That should retrieve all data within the server, and should work regardless of the number of submissions you have uploaded.
```
Comment (adammichaelwood) [inline aggregate-limitations.rst]: where is the link to the Briefcase applet?

Comment (ankita240796) [reply aggregate-limitations.rst]: @adammichaelwood I meant to say here that the server shows the link at the top.
selection_001
Do I need to insert the link here?

Comment (adammichaelwood) [reply aggregate-limitations.rst]: Putting the link here in the docs would be a good idea.

Comment (ankita240796) [reply aggregate-limitations.rst]: The link is an internal link. It opens up an ODK Briefcase page and generates a token. I think it is better to insert an image here.
briefcase-applet

Comment (ankita240796) [reply aggregate-limitations.rst]: @adammichaelwood Should I insert the image ?

**Comment by lognaturel** [comment]
@ankita240796 Wow, it looks like you installed Aggregate 0.9, right?? Given how old it is and that it's no longer supported by AppEngine (as described in https://github.com/opendatakit/briefcase/pull/208#issuecomment-332957723) I wonder whether we should just remove any mention of it?

**Comment by ankita240796** [comment]
Yes @lognaturel , I installed Aggregate 0.9.8.1 and I launched a Test Server to check this. I think that the 1000 record limit is a limitation and anyone using Aggregate 0.9.8.1 should be informed about that. Yes, but we can remove this from this doc and add it to another doc about upgrading Aggregate which I am working on. What do you think?

File: aggregate-limitations.rst (hunk 1 above)
Comment (adammichaelwood) [reply aggregate-limitations.rst]: yes

**Comment by adammichaelwood** [comment]
> I think we can remove this reference from this doc and add it to another doc about upgrading Aggregate which I am working on. What do you think?

I agree with this. We don't want to encourage anyone to install new instances of that version, but helping people with legacy installs upgrade is a great thing to do.

**Comment by ankita240796** [comment]
@adammichaelwood Sure I will remove this from here.

**Comment by ankita240796** [comment]
@adammichaelwood I removed details about Aggregate 0.9.x and resolved the merge conflict too.

**Pull request merged**